core/plugins/org.polarsys.capella.core.platform.sirius.ui.perspective/plugin.xml

```diff
@@ -523,9 +523,39 @@
             pattern="org.polarsys.capella.core.platform.sirius.ui.project/capella.project.wizard">
       </activityPatternBinding>
 
-
-
-
+      <!-- Component deployment concept -->
+      <activityPatternBinding
+            activityId="org.polarsys.capella.activities.componentDeployment"
+            isEqualityPattern="true"
+            pattern="pa.pab.deploy.behavior.pc">
+      </activityPatternBinding>
+      <activityPatternBinding
+            activityId="org.polarsys.capella.activities.componentDeployment"
+            isEqualityPattern="true"
+            pattern="pa.pab.manage.behavior.pcs.deployment">
+      </activityPatternBinding>
+      <activityPatternBinding
+            activityId="org.polarsys.capella.activities.componentDeployment"
+            isEqualityPattern="true"
+            pattern="pa.pab.deployed.behaviour.pcs">
+      </activityPatternBinding>
+      <activityPatternBinding
+            activityId="org.polarsys.capella.activities.componentDeployment"
+            isEqualityPattern="true"
+            pattern="pa.pab.all.deployed.behaviour.pcs">
+      </activityPatternBinding>
+      <activity
+            description="Concepts related to component deployment"
+            id="org.polarsys.capella.activities.componentDeployment"
+            name="Component Deployment Concepts">
```

Comment:
Behavior Deployment Concepts
or maybe handle all kind of deployments iun this category idk
pa.pab.all.deployed.*.pcs, pa.pab.deploy.*.pc,  etc

Reply:
2 new activity categories have been added: Behavior deployment concepts and Node deployment concepts